integration_tests/src/main/python/get_json_test.py

```diff
@@ -99,7 +103,8 @@ def test_get_json_object_spark_unit_tests(query):
             ['{"big": "' + ('x' * 3000) + '"}']]
     assert_gpu_and_cpu_are_equal_collect(
```

Comment:
Why are we not verifying that this falls back to the CPU as expected instead? If we have to divide the test up so we know exactly which once fall back and which ones do not I am fine with that.